cordova-lib/src/plugman/fetch.js

```diff
@@ -330,7 +330,7 @@ function copyPlugin(pinfo, plugins_dir, link) {
 
     shell.rm('-rf', dest);
 
-    if(!link && dest.indexOf(path.resolve(plugin_dir)) === 0) {
+    if(!link && dest.indexOf(path.resolve(plugin_dir)+'\\') === 0) {
```

Comment:
Wouldn't this fail on macs/unix?

with this change, on windows it would be `c:\work\SimplePlugin\`, but on mac it would be `~/work/SimplePlugin\`